docs/src/main/tut/Injection.md

```diff
@@ -47,6 +47,13 @@ implicit val dateToLongInjection = new Injection[java.util.Date, Long] {
 }
 ``` 
 
+We can be less verbose using the `Injection.apply` function:
```

Comment:
If they need to define many of them, it's actually much better to use this method. I think it's ok to show it here. They can choose if they want to type the entire thing or use the simplified factory method.